engine/render/window/Window.cpp

```diff
@@ -28,13 +29,21 @@ bool Window::CreateWindowSurface(VkInstance instance, VkSurfaceKHR* surface)
     return glfwCreateWindowSurface(instance, window, nullptr, surface) == VK_SUCCESS;
 }
 
+Utils::MHArray<const char*> Window::GetRequiredExtensions()
+{
+    uint32_t glfwExtensionCount = 0;
+    const char** glfwExtensions;
+    glfwExtensions = glfwGetRequiredInstanceExtensions(&glfwExtensionCount);
+    return Utils::MHArray<const char*>(glfwExtensions, glfwExtensionCount);
+}
+
 void Window::ResizeCallback(GLFWwindow* windowPtr, int width, int height)
 {
-    auto window = reinterpret_cast<Window*>(glfwGetWindowUserPointer(windowPtr));
+    auto windowPointer = reinterpret_cast<Window*>(glfwGetWindowUserPointer(windowPtr));
```

Comment:
For reference, I would always avoid naming a variable's type as part of its name. It may change, and in this case provides 0 additional info.

Reply:
Changed this to `WindowContext`